live-hls.js

```diff
@@ -606,16 +675,68 @@ master = function(request, response) {
  */
 
 event = function(request, response) {
+
```

Comment:
`event()` and `live()` do much of the same work now. We should try and consolidate this logic to one function have both `event()` and `live()` use that function.